enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -1596,3 +1598,24 @@
       (met/with-gtaps! {:gtaps {:venues (venues-category-native-gtap-def)}, :attributes {"cat" 50}}
         (mt/with-premium-features #{}
           (is (thrown-with-msg? clojure.lang.ExceptionInfo sandboxing-disabled-error (run-venues-count-query))))))))
+
+(def ^:private long-names-dataset
+  (tx/dataset-definition "long-column-names"
+                         [["long_table"
+                           [{:field-name "column_name_with_an_incredibly_verbose_and_exceedingly_detailed_description_that_never_seems_to_end"
+                             :base-type :type/Text}
+                            {:field-name "second_column_name_that_is_even_more_over_the_top_with_its_unnecessarily_expansive_wording"
+                             :base-type :type/Text}]
+                           [["First row, first column" "First row, second column"]
+                            ["Second row, first column" "Second row, second column"]]]]))
+
+(deftest long-column-names-sandbox-native-query-test
```

Comment:
maybe add a test that verified that a query with a long name actually returns less data?